chop/constraints.py

```diff
@@ -281,12 +297,11 @@ class L2Ball(LpBall):
 
     @torch.no_grad()
     def prox(self, x, step_size=None):
```

Comment:
missing docstring. You can probably copy / paste from another prox but its important that all functions have a docstring so that when you search for it in the documentation you get some information.